direct/src/showbase/ShowBase.py

```diff
@@ -289,11 +292,11 @@ def __init__(self, fStartDirect=True, windowType=None):
         #: traverse it automatically in the collisionLoop task, so you won't
         #: need to call :meth:`~panda3d.core.CollisionTraverser.traverse()`
         #: yourself every frame.
-        self.cTrav = 0
-        self.shadowTrav = 0
+        self.cTrav: Optional[CollisionTraverser] = None
+        self.shadowTrav: Optional[CollisionTraverser] = None
         self.cTravStack = Stack()
         # Ditto for an AppTraverser.
-        self.appTrav = 0
+        self.appTrav: Optional[CollisionTraverser] = None
```

Comment:
I don't think appTrav is meant to be a CollisionTraverser, but I don't know what it's meant to be - I would suggest leaving this Any for now